src/core/store/common/tokensByAsset.selectors.ts

```diff
@@ -62,15 +62,16 @@ export const selectDepositTokenOptionsByAsset = createSelector(
             const allowancesMap = userTokensAllowancesMap[address] ?? {};
             return createToken({ tokenData, userTokenData, allowancesMap });
           });
-        let subgraphTokens: TokenView[] = supportedTokens
+        const subgraphTokens: TokenView[] = supportedTokens
           .filter((address) => !!tokensMap[address])
           .map((address) => {
             const tokenData = tokensMap[address];
             const userTokenData = userTokensMap[address];
             const allowancesMap = userTokensAllowancesMap[address] ?? {};
             return createToken({ tokenData, userTokenData, allowancesMap });
           });
-        return unionBy(mainTokens, subgraphTokens, (o) => o.symbol);
+        const sortedSubgraphTokens = sortBy(subgraphTokens, (o) => o.symbol);
```

Comment:
What's o suppose to mean, can we name this or leave a comment?

Reply:
added comment